docs(api): Update listed pipette flow rates

jwwojak opened the pull request:
# Overview

The API docs section on [default pipette flow rates](https://docs.opentrons.com/v2/pipettes/characteristics.html#pipette-flow-rates) shows inaccurate flow rates. The parent page/file is `characteristics.rst`. This updates that table based on the flow rate data provided in these 2 JIRA issues:

- RTC-667
- RESC-375

Both include a list of default flow rates. 

Excluding 20 uL tips because those are not available yet.

## Test Plan and Hands on Testing

Check the updated table to make sure it reflects the correct default flow rates.

## Changelog

Change updates and revised the table that lists flow rates for various pipette and tip combinations. 

Minor text changes in section text.

## Review requests

See RTC-667 for a list of flow rates. I think I've got these reproduced correctly.

## Risk assessment

Low. Docs change only. Minor table and text change in a single section.

jwwojak committed:
Update characteristics.rst

jwwojak committed:
Update characteristics.rst

jwwojak committed:
Update characteristics.rst

jwwojak committed:
Update characteristics.rst

jwwojak committed:
Update characteristics.rst

jwwojak committed:
Update characteristics.rst

jwwojak committed:
Update characteristics.rst

jwwojak committed:
Update characteristics.rst

emilyburghardt commented on api/docs/v2/pipettes/characteristics.rst:
```diff
@@ -184,27 +184,29 @@ These flow rates will remain in effect until you change the ``flow_rate`` attrib
-      - 1000
-      - 716
-
+Flex pipette flow rates depend on pipette model and tip capacity. Each pipette–tip combination has a default flow rate for aspirating, dispensing, and blowing out liquid. When using a 50 µL pipette, you should only use 50 µL tips.
```
Should this be "depend on pipette model, volume, and tip capacity" ? I guess the 96-channel pipettes are technically separated by model, though...

jwwojak replied:
This is a good observation. I wonder about it too. In this case, "capacity" might be sufficient. Here's what I'm thinking:

- Capacity is the maximum rated amount. It's a constant
- Volume is how much space liquid in the tip fills and can vary. This is a variable.

Sometimes a pipette may have less liquid in it (occupy less volume) than the rated capacity, but the flow rate is the same, I think. So the volume doesn't figure in the flow rate. I'm overthinking this probably.

emilyburghardt replied:
Got it. I think just using capacity works. You're right, it technically covers more ground than "volume" in that it's a more exact definition. I'm more used to using "volume" in everyday science speak, so it threw me for a minute. But volumes are listed clearly in the table. I'm rambling but I think this is fine the way you originally had it!

emilyburghardt commented on api/docs/v2/pipettes/characteristics.rst:
```diff
@@ -184,27 +184,29 @@ These flow rates will remain in effect until you change the ``flow_rate`` attrib
+Flex pipette flow rates depend on pipette model and tip capacity. Each pipette–tip combination has a default flow rate for aspirating, dispensing, and blowing out liquid. When using a 50 µL pipette, you should only use 50 µL tips.
+
++-----------------------------+-------------------+------------------------+
+| Pipette Model               | Tip Capacity (µL) | Default Flow Rate (µL) |
```
One super fussy comment: the OT-2 pipette flow rates section beneath states "Default flow rates are the same across all three actions" for aspirate, dispense, and blowout. Do we want something like this for the Flex section? It was a quick question that popped into my head while reading, although I assumed that was the case since there was only one column.

jwwojak replied:
Yes! Let's revise this to make both intro paragraphs similar. Nice observation. Will revise.

emilyburghardt commented on api/docs/v2/pipettes/characteristics.rst:
```diff
@@ -184,27 +184,29 @@ These flow rates will remain in effect until you change the ``flow_rate`` attrib
++-----------------------------+-------------------+------------------------+
+| Pipette Model               | Tip Capacity (µL) | Default Flow Rate (µL) |
++=============================+===================+========================+
+| 1- and 8-channel (50 µL)    | 50                | 35                     |
```
I really like the new layout for the table!

emilyburghardt reviewed (commented):
Thumbs up from me. Left two super nitpicky comments; feel free to take or leave them.

jwwojak committed:
Merge branch 'edge' into docs-flow-rates-update

jwwojak committed:
Merge branch 'edge' into docs-flow-rates-update

jwwojak committed:
Revised based on review feedback

Excluding low-volume, 96-channel pipette flow rates. This pipette has not been released at this time. Will have to revisit these flow rates later, after product launch. Note: flow rate numbers are in the JIRA issue itself (for historic rediscovery).

jwwojak committed:
Merge branch 'edge' into docs-flow-rates-update

emilyburghardt reviewed (approved):
Looks good 👍🏻
